src/main/java/uk/gov/hmcts/darts/cases/controller/CaseController.java

```diff
@@ -70,8 +70,8 @@ public ResponseEntity<List<ScheduledCase>> casesGet(
         LocalDate date
     ) {
         GetCasesRequest request = new GetCasesRequest();
-        request.setCourthouse(courthouse.toUpperCase(Locale.ROOT));
-        request.setCourtroom(courtroom.toUpperCase(Locale.ROOT));
+        request.setCourthouse(DataUtil.toUpperCase(courthouse));
+        request.setCourtroom(DataUtil.toUpperCase(courtroom));
```

Comment:
Any reason not to use the same pattern as you've used in `AudioController` and in `casesPost`? I.e create a static void method in DataUtil that preProcesses?

Reply:
This is creating a brand new object so we create it here with the correct values instead of consuming an already created object